ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,39 +35,209 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
-    }
+  function  assertAddTermCommon(assert, component) {
+    /** Common tests for Add terms, use in term and taxonomy components */
+    // check term title
+    var listOfLinks = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        component,
+        'a'
+    );
+    assert.equal(listOfLinks.length, 2);
+    var linkHeader = listOfLinks[0].getDOMNode();
+    assert.equal(linkHeader.innerHTML, vocabulary.name);
+    //test items
+    var devItems = React.addons.TestUtils.
+      findRenderedDOMComponentWithClass(
+        component,
+        'panel-body'
+    );
+    var itemList = React.addons.TestUtils.
+      scryRenderedDOMComponentsWithTag(
+        devItems,
+        'li'
+    );
+    assert.equal(itemList.length, 2);
+    return devItems;
+  }
+
+  function assertAddVocabularyCommon(assert, component) {
+    /** Common tests for Add Vocabulary, use in term and taxonomy components */
```

Comment:
JSDoc comments go before the function definition, like Javadoc comments. You also need to add information about the parameters and the return value. For example (from the JSDoc site):


    /**
     * Represents a book.
     * @constructor
     * @param {string} title - The title of the book.
     * @param {string} author - The author of the book.
     */
    function Book(title, author) {
    }